lib/investec_open_api/models/transaction.rb

```diff
@@ -22,8 +23,8 @@ def initialize(params)
       set_id
     end
 
-    # At this point, there is no unique identifier being returned from Investec's API.
-    # This method serves to create a stable unique identifier based on the transaction details.
+    # Investec exposes a uuid for uniqueness so this is no longer needed (non-pending transactions).
+    # It is kept here for legacy purposes. It is also useful for pending transactions.
```

Comment:
It may be useful to point out that the UUID is a composite natural key, to prevent surprise, and isn't unique if any of the inputs change: 

_Added a uuid property to the transactions endpoint. ( accountId.Substring(accountId.Length - 5) + postingDate.Replace("-", "") + postedOrder.PadLeft(7, '0') ) This will only be populated for posted transaction on Private Bank Accounts. This is not a backend banking generated ID and will change if any of the properties making it up changes._

Reply:
@stevebissett I've added a description to the `uuid` field.